10318.xml

```diff
@@ -0,0 +1,189 @@
+        <Units>%</Units>
+        <Description><![CDATA[Max CPU Usage of the RCU.]]></Description>
+      </Item>
+      <Item ID="3">
+        <Name>Memory Usage</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
+        <Description><![CDATA[Memory Usage of the RCU.]]></Description>
+      </Item>
+      <Item ID="4">
+        <Name>Storage Usage</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
+        <Description><![CDATA[Storage Usage of the RCU.]]></Description>
+      </Item>
+	  
+	  <Item ID="51">
+        <Name>Battery Level</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
+        <Description><![CDATA[The Battery Level of the RCU.]]></Description>
+      </Item>
+      <Item ID="52">
+        <Name>Network Bandwidth</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>Mbps</Units>
+        <Description><![CDATA[Network Bandwidth of the RCU.]]></Description>
+      </Item>
+      <Item ID="53">
+        <Name>Mobile Signal</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units/>
+        <Description><![CDATA[Mobile Signal of the RCU.]]></Description>
+      </Item>
+      <Item ID="54">
+        <Name>GPS Signal</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units/>
+        <Description><![CDATA[GPS Signal of the RCU.]]></Description>
+      </Item>
+      <Item ID="55">
+        <Name>Wi-Fi Signal</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units/>
+        <Description><![CDATA[Wi-Fi Signal of the RCU.]]></Description>
+      </Item>
+      <Item ID="56">
+        <Name>UpLink Rate</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>Mbps</Units>
```

Comment:
Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

Reply:
Have commented before.

Reply:
Mbps：megabit per second is a unit of data transfer rate.